lms/djangoapps/discussion_api/serializers.py

```diff
@@ -290,6 +290,15 @@ class CommentSerializer(_ContentSerializer):
 
     non_updatable_fields = NON_UPDATABLE_COMMENT_FIELDS
 
+    def __init__(self, *args, **kwargs):
+        remove_fields = kwargs.pop('remove_fields', None)
```

Comment:
It looks like there are potential subtle differences in behavior between the current pattern and what I'm proposing.  Let's play it safe, and leave this alone.